model/src/w3idatmd.F90

```diff
@@ -257,6 +270,16 @@ MODULE W3IDATMD
 !/
       CONTAINS
 !/ ------------------------------------------------------------------- /
+!>
+!> @brief Set up the number of grids to be used.
+!>
+!> @details Use data stored in NGRIDS in W3GDATMD.
+!>
+!> @param[in] NDSE
```

Comment:
The variable definitions are: 
!       NDSE    Int.   I   Error output unit number.
! NDST Int. I Test output unit number.
Can we add that here?

Reply:
Yes!  Good catch.  I'll make that update.